packages/e2e/src/specs/overview/mc_version_check.e2e.ts

```diff
@@ -0,0 +1,40 @@
+import NavigationTabs from "../../page-objects/tabs.page"
+import settingsPage from "../../page-objects/settings.page"
+import { version } from "../../../../app/package.json"
+
+describe("Navigation Menu check ", () => {
+
+    before(async () => {
+      // Waiting for device connected through USB
+      await browser.executeAsync((done) => {
+        setTimeout(done, 10000)
+      })
+    })
+
+      it("Check Mudita Center version - app. vs. package.json", async () => {
```

Comment:
Maybe to follow the convention (after taking a look on other `.e2e.ts` files) it would be good to change to something like
`it("should have the same Mudita Center version in app as in package.json")`?